numpy/fft/tests/test_pocketfft.py

```diff
@@ -38,7 +38,7 @@ def test_identity_long_short(self, dtype):
         # Test with explicitly given number of points, both for n
         # smaller and for n larger than the input size.
         maxlen = 16
-        atol = 4 * np.spacing(np.array(1., dtype=dtype))
+        atol = 5 * np.spacing(np.array(1., dtype=dtype))
```

Comment:
Good, of course - this was a definitely one of those trial-and-error numbers (with √16 seeming reasonable...)

Reply:
It has only been a problem with musl. I suspect inaccuracies in the long double sin/cos functions.